### ISSUE TITLE
cookies

### ISSUE THREAD

**Issue by bsureshkrishna** [m2b3/SciCommons-frontend#78]
So as now, the site does not use cookies at all, is that right ?

**Comment by dinakar17** [comment]
Yes, I updated the code last week in which cookies are used to store the access token (the token that is created after a user logs in).

**Comment by bsureshkrishna** [comment]
and after your update, cookies are not used any more ? how does it work then ?

**Comment by dinakar17** [comment]
Cookies are used after the update. In other words, the current website uses cookies to store tokens.

**Comment by bsureshkrishna** [comment]
So then we **do** have an issue with cookie permissions, gdpr, etc. I assume these are non-tracking cookies.. but we need some sort of permissions check.

**Comment by bsureshkrishna** [comment]
this should be a feature-request. for now, it is not a bug.
